compose/compose-file/compose-file-v3.md

```diff
@@ -1397,7 +1397,7 @@ the alias, or the service name if no alias was specified.
 
 Links are not required to enable services to communicate - by default,
 any service can reach any other service at that service’s name. (See also, the
-[Links topic in Networking in Compose](../networking.md#links).)
+[Links topic in Networking in Compose](../networking.md#link-containers).)
```

Comment:
not important: I am not a fan of referring to sections as "topics". I think it doesn't make sense unless you're a technical writer :D

```suggestion
[Link containers section in Networking in Compose](../networking.md#link-containers).)
```

Reply:
You're not wrong - changing now!